src/integration-test/java/com/commercetools/sync/integration/types/TypeSyncIT.java

```diff
@@ -0,0 +1,892 @@
+package com.commercetools.sync.integration.types;
```

Comment:
We usually have as a convention the integration tests in two packages:
1. `/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/{resource name}/`
2. `/src/integration-test/java/com/commercetools/sync/integration/externalsource/{resource name}/` 

so for types it should be:

1. `/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/types/`: here you should have ITs which are related to syncing from a source to a target CTP project.
2. `/src/integration-test/java/com/commercetools/sync/integration/externalsource/types/`: here you should have ITs which are related to syncing from a source feed to a target CTP project.


This is useful also for users as a convention to know where to find examples of their corresponding use cases.

Reply:
Nice catch! I think I should also `ctpprojectsource`tests for the `Type` resource. Should we open this as a new issue or do in this branch, WDYT? I moved already existing IT's to `externalsource` package. Ready to review:  https://github.com/commercetools/commercetools-sync-java/pull/333/commits/3151401474c5b52b80bedb4642f29a310dddb8cb